### ISSUE TITLE
[DomCrawler] disabled form fields not reachable anymore

### ISSUE THREAD

**Issue by dmaicher** [symfony/symfony#35923]
**Symfony version(s) affected**: 3.4.38, 4.4.5, 5.0.5

**Description**  
It seems since https://github.com/symfony/symfony/pull/34059 a disabled form field is not added to the form anymore at all which seems odd.

In one of my tests I was retrieving a disabled form field and checking its value. This is not possible anymore now.

**How to reproduce**  
```php
<?php

use Symfony\Component\DomCrawler\Crawler;

$html = <<<HTML
<html>
    <form>
        <input type="text" name="foo" value="bar" disabled />
    </form>
</html>
HTML;

$crawler = new Crawler($html, 'http://localhost/');

dump($crawler->filter('form')->form()->get('foo')->getValue());
```

Works with 4.4.4 and breaks with 4.4.5: `InvalidArgumentException: Unreachable field "foo"`

**Comment by alexpott** [comment]
The change in #34059 has broken the Drupal test suite too.

We have code like
```
    $this->assertSession()->checkboxChecked('options[revision]');
    $this->assertSession()->fieldDisabled('options[revision]');
```
The call to checkboxChecked ends up throwing an error `InvalidArgumentException: Unreachable field "revision"`

I think that the change in #34059  is too broad.

**Comment by nicolas-grekas** [comment]
We should revert #34059 for sure.

> I think that the change in #34059 is too broad.

"too broad" => what's the reduced scope you've in mind? Up for a PR?

**Comment by dmaicher** [comment]
https://github.com/symfony/symfony/pull/35937 reverts it for now until there is a "better" fix for https://github.com/symfony/symfony/issues/28179

**Comment by VincentLanglet** [comment]
Shouldn't a `4.4.6` be released to fix the BC-break ?

**Comment by dmaicher** [comment]
> Shouldn't a `4.4.6` be released to fix the BC-break ?

There will be a new monthly patch release in 3 weeks or so. Possibly earlier... not sure.

In the meantime just fix `symfony/dom-crawler` to 4.4.4 :wink:
